nautobot/dcim/filters.py

```diff
@@ -1151,7 +1102,7 @@ def search(self, queryset, name, value):
         return queryset.filter(qs_filter)
 
 
-class CableFilterSet(StatusModelFilterSetMixin, BaseFilterSet, CreatedUpdatedFilterSet, CustomFieldModelFilterSet):
+class CableFilterSet(StatusModelFilterSetMixin, NautobotFilterSet):
```

Comment:
Is the order here significant in some way or can it be changed to `(NautobotFilterSet, StatusModelFilterSetMixin)` to be consistent with the others?

Reply:
Good spot